Twig/Extension/CKEditorExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Sonata\FormatterBundle\Twig\Extension;
+
+use Ivory\CKEditorBundle\Templating\CKEditorHelper;
+
+/**
+ * CKEditorExtension.
+ *
+ * @author Marodon Jérémy <marodon.jeremy@gmail.com>
+ */
+class CKEditorExtension extends \Twig_Extension
+{
+    /** @var \Ivory\CKEditorBundle\Templating\CKEditorHelper */
+    private $helper;
+
+    /**
+     * Creates a CKEditor extension.
+     *
+     * @param \Ivory\CKEditorBundle\Templating\CKEditorHelper $helper The CKEditor helper.
```

Comment:
FQN = Full Qualified Name.

Basicly, import classes with `use` and write only the class name.